frontend/.storybook/mockData.ts

```diff
@@ -0,0 +1,85 @@
+import { auditLogFactory } from "../src/fixtures/auditLog";
+import { destructionListAssigneesFactory } from "../src/fixtures/destructionList";
+import { FIXTURE_SELECTIELIJSTKLASSE_CHOICES } from "../src/fixtures/selectieLijstKlasseChoices";
+import { userFactory, usersFactory } from "../src/fixtures/user";
+import { zaaktypeChoicesFactory } from "../src/fixtures/zaaktypeChoices";
+
+export const MOCKS = {
+  AUDIT_LOG: {
+    url: "http://localhost:8000/api/v1/destruction-lists/00000000-0000-0000-0000-000000000000/auditlog/?",
```

Comment:
I think that we can remove the `?`?

Reply:
- [ ] Test if question mark can be removed from mock